openlibrary/plugins/recaptcha/recaptcha.py

```diff
@@ -51,7 +52,12 @@ def validate(self, value=None):
             response=i.recaptcha_response_field,
             remoteip=web.ctx.ip)
 
-        response = urllib.urlopen('http://www.google.com/recaptcha/api/verify', urllib.urlencode(data)).read()
+        # If HTTP call fails, let the user through
+        try:
+            response = urllib2.urlopen('http://www.google.com/recaptcha/api/verify', urllib.urlencode(data), timeout=2).read()
+        except urllib2.URLError:
```

Comment:
urllib2.HTTPError is a subclass of urllib2.URLError, so should be okay there.

That said, good point about abuse. My choice implies that we think the Google API call will fail more often than people will hack the captcha. I think this choice is okay for now, but let's keep an eye on it.